src/platform/test/functional/page_objects/home_page.ts

```diff
@@ -91,8 +98,7 @@ export class HomePageObject extends FtrService {
 
   async addSampleDataSet(id: string) {
     await this.openSampleDataAccordion();
-    await this.retry.waitFor('sample data to be installed', async () => {
-      // count for the edge case where some how installation completes just before the retry occurs
```

Comment:
removed the comment, because this condition guards the scenario where the sample data was installed by a previous test